Module/src/PSWordCloud/NewWordCloudCommand.cs

```diff
@@ -945,7 +991,7 @@ private static IEnumerable<string> SortWordList(IDictionary<string, float> dicti
             }
 
             Complex point;
-            float angle = 0, maxAngle = 0, angleIncrement = 360 / (radius / 6 + 1);
+            float angle = 0, angleIncrement = 360 / (radius / 6 + 1);
```

Comment:
nit: Probably break this out into separate lines.  Especially if PowerShell folks are going to be working in the repo, this looks like an array literal and is hard to visually parse.